features/603-asn-update-for-HO100301-nextui/test.feature

```diff
@@ -0,0 +1,22 @@
+Feature: 605 - ASN Update / Correction in the Next UI when HO100301 raised
+
+			"""
+			There is an exception (HO100301 - ASN not found on PNC) on the incoming message, which enables user to edit ASN. When fixed, shows the ASN correction.
+			"""
+
+	Background:
+		Given the data for this test is not in the PNC
+			And "input-message" is received
+
+	@NextUI
```

Comment:
For consistency, I think we should use white-space characters instead tab characters for indentation in `.feature` files.